update-dois/src/main/java/org/reactome/release/updateDOIs/Main.java

```diff
@@ -59,13 +58,12 @@ public static void main( String[] args ) throws IOException {
       // Set up db connections.
       dbaTestReactome = new MySQLAdaptor(hostTR, databaseTR, userTR, passwordTR, portTR);
       dbaGkCentral = new MySQLAdaptor(hostGK, databaseGK, userGK, passwordGK, portGK);
-    } catch (Exception e) {
-      e.printStackTrace();
-      System.exit(1);
+    } catch (SQLException e) {
+      throw new SQLException("Unable to create MySQL adaptor. Check properties file.");
```

Comment:
Since the SQLException is thrown in the method signature, the try/catch block can be removed entirely.